tests/test_subtractTask.py

```diff
@@ -155,6 +155,9 @@ def test_config_validate_forceCompatibility(self):
         config = subtractImages.AlardLuptonSubtractTask.ConfigClass()
         config.mode = "auto"
         config.forceCompatibility = True
+        with self.assertRaises(ValueError):
+            config.validate()
+        config.mode = 'convolveTemplate'
```

Comment:
Here's what I'd done (after the `forceCompatibility=True` line), and removing the two lines below:

```
        # Ensure we're not trying to change config values inside validate().
        config.freeze()
        with self.assertRaisesRegex(FieldValidationError,
                                    "forceCompatibility=True requires mode='convolveTemplate'"):
            config.validate()

        config = subtractImages.AlardLuptonSubtractTask.ConfigClass()
        config.mode = "convolveTemplate"
        config.forceCompatibility = True
        # Ensure we're not trying to change config values inside validate().
        config.freeze()
        # Should not raise:
        config.validate()
```